source/_docs/energy/water.markdown

```diff
@@ -18,30 +26,28 @@ We have the following integrations available for existing products that can prov
 - [Flo](/integrations/flo)
 - [Flume](/integrations/flume)
 - [HomeWizard Energy](/integrations/homewizard)
-- [P1 Monitor](/integrations/p1_monitor)
 - [StreamLabs](/integrations/streamlabswater)
 - [Suez Water](/integrations/suez_water)
 
-There are also products for water usage monitoring that are based on existing common IoT protocol standards:
+## Community Made Sensors
+If your water meter lacks a rotary disk, magnetic disk, or coil. There are alternative solutions available to seamlessly integrate water monitoring into your smart home setup:
+- [AI-on-the-edge-device](https://github.com/jomjol/AI-on-the-edge-device) is a project running on an ESP32-CAM and can be fully integrated into Home Assistant using the Home Assistant Discovery Functionality of MQTT. It digitalizes your gas/water/electricity meter display and provides its data in various ways.![Photo of the AI-on-the-edge-device Workflow](/images/docs/energy/
 
-- [Z-Wave](/integrations/zwave_js)
```

Comment:
why are these removed?

Reply:
I removed the Z-Wave details because they shift focus from our main narrative to technical specifics, better suited for a DIY guide. This keeps our content focused and relevant.

Reply:
This is not a DIY-guide.
As a matter of fact, non-DIY should be the main focus.

Please restore these, and put them above the DIY solutions.